cmd/ferretdb/main.go

```diff
@@ -125,6 +126,14 @@ var cli struct {
 			Package        string        `default:""                             help:"Telemetry: custom package type."`
 		} `embed:"" prefix:"telemetry-"`
 	} `embed:"" prefix:"test-"`
+
+	OTEL struct {
```

Comment:
Flags are named after `OTEL_*` env variables for better consistency: https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/.

Reply:
> It should be configurable with `--test-XXX` flags.

They should be under the Test group